trikot-viewmodels-declarative-flow/sample/android/src/main/java/com/mirego/sample/ui/BaseSampleActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.mirego.sample.ui
+
+import android.os.Bundle
+import androidx.activity.enableEdgeToEdge
+import com.mirego.trikot.viewmodels.declarative.controller.VMDNavigationDelegate
+import com.mirego.trikot.viewmodels.declarative.controller.VMDViewModelController
+import com.mirego.trikot.viewmodels.declarative.controller.ViewModelActivity
+import com.mirego.trikot.viewmodels.declarative.viewmodel.VMDViewModel
+
+abstract class BaseSampleActivity<VMC : VMDViewModelController<VM, N>, VM : VMDViewModel, N : VMDNavigationDelegate> :
+    ViewModelActivity<VMC, VM, N>() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        enableEdgeToEdge()
```

Comment:
I don't know if it really matters, but in the doc they call the method _before_ calling super()